main.tex

```diff
@@ -1,9 +1,13 @@
 \documentclass[english]{notes}
 
-\usepackage{./mama}
+
 \usepackage{blkarray}
 \usepackage{wrapfig}
 \usepackage{graphicx}
+\usepackage{subfiles}%Allows compilation of subfiles.
+\usepackage{./mama}%keep order of mama and subfiles packages
```

Comment:
Style convention: put comments above the commented line, like this:
```
% allow compilation of subfiles.
\usepackage{subfiles}
```
Notice the space after `%` (sorry to be this picky!)